spec/support/controller_macros.rb

```diff
@@ -27,5 +27,8 @@ def api_request_headers(developer, user)
     request.headers['X-Api-Key'] = developer.api_key
     request.headers['X-User-Token'] = user.authentication_token
     request.headers['X-User-Email'] = user.email
+
+    request.headers['HTTP_AUTHORIZATION'] =
```

Comment:
need to pass token header for api requests since we now using new [before_action](https://github.com/earlymarket/Coposition/pull/662/files#diff-0ef0779741ad3209b46314a66c353351R7)